prowler/providers/azure/services/network/network_service.py

```diff
@@ -0,0 +1,131 @@
+from dataclasses import dataclass
+
+import requests
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.network.models import NetworkWatcher
+
+from prowler.lib.logger import logger
+from prowler.providers.azure.lib.service.service import AzureService
+
+
+########################## SQLServer
+class Network(AzureService):
+    def __init__(self, audit_info):
+        super().__init__(NetworkManagementClient, audit_info)
+        self.token = self.__get_token__(audit_info)
+        self.security_groups = self.__get_security_groups__(self.token)
+        self.bastion_hosts = self.__get_bastion_hosts__()
+
+    def __get_security_groups__(self, token):
+        logger.info("SQL Server - Getting Network Security Groups...")
+        security_groups = {}
+        for subscription, client in self.clients.items():
+            try:
+                security_groups.update({subscription: []})
+                security_groups_list = client.network_security_groups.list_all()
+                available_locations = {}
+                network_watchers = self.__get_network_watchers__(client, subscription)
+                for security_group in security_groups_list:
+                    subscription_id = security_group.id.split("/")[2]
+                    if subscription_id not in available_locations:
+                        available_locations[subscription_id] = (
+                            self.__get_subscription_locations__(subscription_id, token)
+                        )
+                    subscription_locations = available_locations[subscription_id]
+                    security_groups[subscription].append(
+                        SecurityGroup(
+                            id=security_group.id,
+                            name=security_group.name,
+                            location=security_group.location,
+                            security_rules=security_group.security_rules,
+                            network_watchers=network_watchers,
+                            subscription_locations=subscription_locations,
+                            flow_logs=self.__get_flow_logs__(subscription),
```

Comment:
add it to the new Network Watchers class

Reply:
Done